test/Microsoft.AspNet.Mvc.DataAnnotations.Test/DataAnnotationsModelValidatorTest.cs

```diff
@@ -64,15 +58,15 @@ public static IEnumerable<object[]> Validate_SetsMemberName_OnValidationContext_
             string expectedMemberName)
         {
             // Arrange
-            var attribute = new Mock<ValidationAttribute> { CallBase = true };
-            attribute.Protected()
```

Comment:
Moq.Protected probably works correctly in CoreCLR now (the bug was addressed), but the change looks nicer + is strongly typed.